### PR TITLE
add makesydebib command, urls display in bib

### LINKED ISSUE #8: Reference page incorrectly citing misc reference
example:
14349172_10202233907806409_11899943_n png
should be 
14355947_1800784840198250_94732294_n png

I could be wrong here, but its missing the url....

### PR DESCRIPTION
**Pull request by nolan-cerebras**
Closes #8 
- added a makesydebib command which does

```
\bibliographystyle{unsrt}
\bibliography{references}
```
- urls display in bibliography now
  screen shot 2016-09-16 at 2 47 13 pm

### CODE REVIEW

File: sydestyle.sty [hunk 1]
```diff
@@ -122,4 +122,9 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % End Title Page Stuff
 
+\newcommand*{\makesydebib}{
```
Comment (bcaldwell) [inline sydestyle.sty]: Would be better to take a parameter and pass it into `\bibliography` to support multiple .bib files.
For example you can do `\bibliography{file1,references}`

**Review by bcaldwell — COMMENTED** [review]
https://github.com/Erik-J-D/sydetex/issues/8

**Comment by LaurierMantel** [comment]
@Erik-J-D thoughts?

**Comment by Erik-J-D** [comment]
`howpublished` is the right way to do this, but I think makesydebib is overkill if all it does is call `bibliography`  and `bibliographystyle`, also doesn't do anything syde specific.

**Comment by nolan-cerebras** [comment]
@Erik-J-D it also fixed the text alignment of the bibliography now (just pushed commit)

**Comment by Erik-J-D** [comment]
looks good to me, rebase on master pls and I'll merge

**Comment by nolan-cerebras** [comment]
@Erik-J-D done. Thanks for making this repo btw. Life saver

**Comment by Erik-J-D** [comment]
@ndey96 np. Just noticed you only compiled the report pdf not the wtr as well

**Comment by nolan-cerebras** [comment]
@Erik-J-D updated

**Pull request merged**